build.rs

```diff
@@ -381,8 +384,9 @@ fn pregenerate_asm_main() {
         perlasm(&perlasm_src_dsts, asm_target);
 
         if asm_target.preassemble {
-            // Preassembly is currently only done for Windows targets.
-            assert_eq!(&asm_target.oss, &[WINDOWS]);
+            // Preassembly is currently done for Windows and UEFI targets.
```

Comment:
let's just remove the comment since there's now a `preassemble` field that makes this clear, and it duplicates the assertion below.